pkg/gateway/gateway.go

```diff
@@ -350,11 +352,30 @@ func (g *gatewayModule) isTraefikStarted(z *zinit.Client) (bool, error) {
 	return traefikStatus.State.Is(zinit.ServiceStateRunning), nil
 }
 
+func (g *gatewayModule) traefikBinary(ctx context.Context) (string, string, error) { // path, name, err
+	var exec struct{ Exec string }
+	f, err := os.Open("/etc/zinit/traefik.yaml")
```

Comment:
can u move this logic to Zinit lib instead `pkg/zinit/service.go` it already has AddService, and RemoveService then u can do GetService to return a service object.